test-guru/app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,48 @@
+class QuestionsController < ApplicationController
+  before_action :find_test, only: [:index, :create]
+  before_action :find_question, only: [:show, :destroy]
+
+  rescue_from ActiveRecord::RecordNotFound, with: :rescue_with_question_not_found
+
+
+  def index
+    questions = @test.questions.pluck(:text)
+    render plain: questions.join("\n")
+  end
+
+  def show
+    render plain: @question.text
+  end
+
+  def create
+    question = Question.create(question_params)
+    question.test_id = @test.id
+    question.save
+    render plain: question.inspect
+  end
+
+  def new; end
+
+  def destroy
+    @question.delete
```

Comment:
```suggestion
    @question.destroy
```
почти всегда в рельсах используется `destroy`, а не `delete`
первый метод вызывает коллбеки модели (к примеру `dependent: :destroy` у ассоциации), второй вариант просто удаляет объект из базы, можно сказать "мимо рельс", что может приводить к различным нежелательным проблеммам